client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -66,9 +67,24 @@ export const OutboundLineEditTable = ({
     (row: DraftStockOutLineFragment) => {
       if (nonAllocatableLines.some(line => line.id === row.id)) return true;
 
-      // For Outbound Shipments, we also don't allow allocating bad VVM status stock
-      if (prefs.manageVvmStatusForStock && item?.isVaccine)
-        return !!row.vvmStatus?.unusable;
```

Comment:
Need to watch for this kind of thing -- was sometimes returning early even though it still needed to check other cases. Now it only returns if it actually meets *this* criteria